_docs/_latest/configure/scanner/rule-engine.md

```diff
@@ -19,6 +19,8 @@ for the following resources:
 For examples, see the [rules](https://github.com/GoogleCloudPlatform/forseti-security/tree/dev/rules) 
 directory.
 
+---
+
```

Comment:
This document actually isn't a configure document and should be moved elsewhere (develop?) or deleted.